install_pim/docker/installation_docker.rst

```diff
@@ -133,7 +133,7 @@ Akeneo PIM is configured with environment variables. The `Symfony documentation
 Running jobs
 ************
 
-Akeneo PIM implements a queue for the `jobs <../../import_and_export_data/index.html>`_, as a PHP daemon. This daemon is a Symfony command, that can only execute one job at a time. It does not consume any other job until the job is finished.
+Akeneo PIM implements a queue for the `jobs <../../import_and_export_data/index.rst>`_, as a PHP daemon. This daemon is a Symfony command, that can only execute one job at a time. It does not consume any other job until the job is finished.
```

Comment:
```suggestion
Akeneo PIM implements a queue for the :doc:`/import_and_export_data/index` jobs, as a PHP daemon. This daemon is a Symfony command, that can only execute one job at a time. It does not consume any other job until the job is finished.
```

Side note: this is obsolete since 5.0